.agents/skills/cuopt-server-api-python/assets/pdp_basic/client.py

```
@@ -0,0 +1,94 @@
+            "task_locations": [1, 2, 3, 4],
+            "demand": [[10, -10, 15, -15]],
+            "pickup_and_delivery_pairs": [[0, 1], [2, 3]],
+        },
+        "fleet_data": {
+            "vehicle_locations": [[0, 0]],
+            "capacities": [[50]],
+        },
+        "solver_config": {"time_limit": 10},
+    }
+
+    response = requests.post(
```

Ruff (0.15.17)

[error] Probable use of `requests` call without timeout

(S113)